core/app/models/spree/user_last_url_storer.rb

```diff
@@ -28,15 +28,6 @@ def initialize(controller)
       @controller = controller
     end
 
-    # Stores into session[:spree_user_return_to] the request full path for
-    # future redirects (to be used after successful authentication). When
-    # there is a rule match then the request full path is not stored.
-    def store_location
```

Comment:
Unfortunately, we can't just remove public methods from classes since people may be still using them. A better approach could be deprecating the method in order to alert people that we are going to remove this method in the next major version. I think that `store_location` is the only method used so we can probably deprecate the whole class usage.